CharacterCreation/04_Backgrounds.txt

```diff
@@ -0,0 +1,247 @@
+==============================
+Backgrounds
+==============================
+	Where is your character from? All characters have a past: where they grew up, 
+what career they studied or were trained for, what languages they speak and 
+understand. A character's background will influence their patterns of behavior, 
+ability to communicate with others, their world-view, their areas of specialized 
+knowledge, and more. Maybe they grew up on a kangaroo farm, or studied medicine 
+on a core planet. Were they trained for corporate espionage or recruited for 
+military service? From smuggling contraband, to piloting cargo ships, to making 
+appointments for a corporate executive, a character's past will inform how they 
+act under pressure, react under fire and interact with teammates.
+
+	In addition to influencing your character's decisionmaking and roleplay, 
```

Comment:
nitpick: the characters aren't roleplaying; they're just living. The player is roleplaying. Reword?